recipe/meta.yaml

```diff
@@ -1,60 +1,91 @@
 {% set name = "econml" %}
-{% set version = "0.13.1" %}
+{% set version = "0.15.0" %}
 
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/econml-{{ version }}.tar.gz
-  sha256: 9060e54f46657a62c67e26a6755feb0531106b24e7444fa4c86b8139c89cf9b9
+  url: https://github.com/py-why/EconML/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: bdb43db013b041253872424105262c62a058783b1ed7cbb6a3890061ec4162d5
+  patches:
+    - patches/0001-do-not-use-econml.tests.patch
+    - patches/0002-np.float-deprecated-alias-use-builtin-float.patch
 
 build:
   number: 0
-  script: {{ PYTHON }} -m pip install . -vv
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  skip: True  # [s390x or py>311]
+  # Segfault detected on some platforms during testing. This is enough of a reason to skip this arch for now
+  # Unfortunately this segfault is not documented anywhere
+  skip: True  # [linux and aarch64]
+  skip: True  # [osx]
 
 requirements:
   build:
     - {{ compiler('c') }}
-    - {{ compiler('cxx') }}
-    - python                                 # [build_platform != target_platform]
-    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
-    - cython                                 # [build_platform != target_platform]
-    - numpy                                  # [build_platform != target_platform]
+    - patch       # [not win]
+    - m2-patch    # [win]
   host:
-    - cython
+    - cython 0.29.36
+    - scipy
     - numpy
     - pip
     - python
+    - wheel
+    - setuptools
   run:
-    - dowhy <0.8
     - lightgbm
-    - python-graphviz
     - joblib >=0.13.0
-    - numba !=0.42.1
     - numpy
-    - pandas
+    - numba
```

Comment:
Could you comment that `numba` is required here https://github.com/py-why/EconML/blob/v0.15.0/econml/solutions/causal_analysis/_causal_analysis.py#L11?